docs/reference/supported-environments.md

```diff
@@ -33,7 +33,25 @@ For example, 1.2+ means support for the minor version 2, and any higher minors (
 
 ## Camunda 8 Self-Managed
 
-We highly recommend running Camunda 8 Self-Managed in a Kubernetes environment. We provide officially supported [Helm Charts](/self-managed/platform-deployment/helm-kubernetes/overview.md) for this. Please follow the [Installation Guide](/self-managed/platform-deployment/overview.md) to learn more about installation possibilities.
+We highly recommend running Camunda 8 Self-Managed in a Kubernetes environment. We provide officially supported [Helm Charts](/self-managed/platform-deployment/helm-kubernetes/overview.md) for this. Follow the [Installation Guide](/self-managed/platform-deployment/overview.md) to learn more about installation possibilities.
+
+The following list includes links for supported Kubernetes environments and recommended starting config. Generally speaking, the specification depends on your needs and workloads.
```

Comment:
@akeller can you please make this more precise (or align with EMs to do so)? Is this the absolute minium requirement you always must have, or is the the "average", but you can be above or below based on workloads?

Reply:
AFAIK, this is average. Minimum would imply a smaller installation footprint (1 Zeebe vs. 3) and possible reduced availability.